File: data/data.code.php.client/AckTestJsOk.php
<?php
namespace protocol;

use network\Packet;


class AckTestJsOk
{
	private $u64 = 0;
	private $i64 = 0;


	public function __construct(&$packet)
	{
		$this->decode($packet);
	}

	public function decode(&$packet)
	{
		$this->u64 = $packet->readU64();
		$this->i64 = $packet->readI64();
	}


	public function getU64()
	{
		return $this->u64;
	}

	public function getI64()
	{
		return $this->i64;
	}

}

File: data/data.code.cpp.client/AckTestJsOk.cpp
#ifndef _ACK_TEST_JS_OK_
#define _ACK_TEST_JS_OK_

#include <list>

#include <pb_type.hpp>
#include <Packet.hpp>

#include <Msg.hpp>



class AckTestJsOk
{
private:
	U64 m_u64;
	I64 m_i64;


public:
	AckTestJsOk(Packet* packet)
	{
		m_u64 = packet->ReadU64();
		m_i64 = packet->ReadI64();
	}


	U64 GetU64()
	{
		return m_u64;
	}

	I64 GetI64()
	{
		return m_i64;
	}

};

#endif
